src/spikeinterface/core/template_tools.py

```diff
@@ -248,3 +249,165 @@ def get_template_extremum_amplitude(
         unit_amplitudes[unit_id] = extremum_amplitudes[unit_id][best_channel]
 
     return unit_amplitudes
+
+
+def move_templates(templates, displacements, source_probe, dest_probe=None, interpolation_method="cubic"):
+    """
+    Move all templates given some displacements using spatial interpolation (cubic or linear).
+    Optionally can be remapped to another probe with a diffrents geometry.
+
+    Before interpolation this function make is checking that templates can be interpolated on new
+    positions using the sparsity information : the entire sparse template must be covered at all new positions.
+    If the Templates object is dense then no check is possible and a wrning is emited.
+
```

Comment:
```suggestion
    Before interpolation this function checks that the templates can be interpolated to new
    positions using the sparsity information: the entire sparse templates must be covered at all new positions.
    If the Templates object is dense then no check is possible and a warning is emitted.

```

covered isn't quite the right word, but I'm still trying to think of a way to write this.